hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/resource/MaintainIndexResource.kt

```diff
@@ -167,7 +167,7 @@ class MaintainIndexResource(private val maintainIndexService: MaintainIndexServi
       `index-housekeeping-cronjob`
       """,
   )
-  fun checkIfComplete() {
+  fun checkIfComplete() =
```

Comment:
the cronjob was previously not returning any output.  Since we've got the index status we might as well return that